app/i18n/fr/gen.php

```diff
@@ -23,6 +23,7 @@
 		'update' => 'Mettre à jour',
 	),
 	'auth' => array(
+		'accept_tos' => 'Vous acceptez les <a href="%s">Conditions Générales d’Utilisation</a>.',
```

Comment:
Maybe "Vous devez accepter..." ?

Reply:
I think it's quite strange to have "Vous devez accepter…" next to the checkbox, but so is "Vous acceptez…" now that I think about it. What about "Accepter…"?

Reply:
Or "J'accepte…"

Reply:
Then the same would apply to English. (I accept the TOS/I agree to the TOS…)

Reply:
Either "Accepter" (my first choice in French), or "J'accepte" (although a bit more "infantilisant" - I cannot find a translation of this concept in English, something like talking to someone who is not fully educated / grown up) sound fine